workspaces/cli-shared/src/diffs/trail-parsers.ts

```diff
@@ -1,70 +1,58 @@
 import { IRequestSpecTrail } from './request-spec-trail';
 import { IInteractionTrail } from './interaction-trail';
 
-// TODO QPB - remove these types - these are the same as the types in ui-v2 - these should be moved to optic-domain
-export interface IPathParameter {
-  id: string;
-  name: string;
-  isParameterized: boolean;
-  description: string;
-  endpointId: string;
-}
-
-interface IEndpoint {
+// This is a subset of IEndpoint in ui-v2
```

Comment:
Creating the minimal amount of data we need that should match IEndpoint so we dont need to copy paste types, and instead, this function specifies the minimum data is needs (if it gets more, it ignores it, the consumers can pass in more)